apps/android/firefox.py

```diff
@@ -74,28 +83,145 @@ def start(self, url="about:blank"):
             }
         )
 
-        device.push(profile.profile, device_profile)
-        device.chmod(device_storage, recursive=True)
+        self.device.push(self.profile.profile, device_profile)
+        self.device.chmod(device_storage, recursive=True)
 
-        app_args = [
-            "-profile",
-            device_profile,
-            "--marionette",
-            "--es",
-            "env0",
-            "LOG_VERBOSE=1",
-            "--es",
-            "env1",
-            "R_LOG_LEVEL=6",
-        ]
+    def run_android_app(self, url):
+        pass
 
-        # start app
-        device.stop_application(self.APP_NAME)
-        device.launch_activity(
+
+class GeckoViewExampleApp(AndroidFirefox):
+    APP_NAME = "org.mozilla.geckoview_example"
+    ACTIVITY_NAME = "GeckoViewActivity"
+
+    def __init__(self, *args, **kwargs):
+        super(GeckoViewExampleApp, self).__init__(*args, **kwargs)
```

Comment:
Do we need to define `__init__` if we're just calling the parent class initialisation method?

Reply:
Note this comment applies to all app sub classes.